packages/framework-integration-tests/integration/providers/aws/subscriptions.integration.ts

```diff
@@ -17,12 +17,11 @@ describe('subscriptions', () => {
     })
 
     it('should delete a subscription when the client calls "unsubscribe"', async () => {
-      const mockCartId = random.uuid()
       const originalSubscriptionsCount = await countSubscriptionsItems()
 
       // Let's create two subscriptions to the same read model
-      cartSubscription(client, mockCartId).subscribe(() => {})
-      const subscriptionObservable = cartSubscription(client, mockCartId).subscribe(() => {})
+      cartSubscription(client, random.uuid()).subscribe(() => {})
```

Comment:
is there any difference between subscribing to an existing cart id vs a non-existing one?

Reply:
It should not. But this was 80% out of my desperation and 20% a suspicion that ApolloClient could be caching this (it should not)